omod-1.9/src/test/java/org/openmrs/module/webservices/rest/web/v1_0/controller/openmrs1_9/PatientIdentifierController1_9Test.java

```diff
@@ -189,4 +191,61 @@ public void shouldReturnTheAuditInfoForTheFullRepresentation() throws Exception
 		
 		assertNotNull(PropertyUtils.getProperty(result, "auditInfo"));
 	}
+
```

Comment:
This seems unrelated to this ticket, was this done to support a different issue?